client/js/screens/Landing/styles.js

```diff
@@ -0,0 +1,19 @@
+import { StyleSheet } from "react-native";
+import { colors, fonts } from "../../config/styles";
+
+const styles = StyleSheet.create({
+  background: {
+    width: '100%',
+    height: '100%' 
+  },
+  logo: {
+    width: 200,
+    height: 100
```

Comment:
use flexbox instead of giving fixed height and width. It will not work on all phones.